client/my-sites/plugins/marketplace/components/marketplace-shopping-cart/index.tsx

```diff
@@ -0,0 +1,74 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { useShoppingCart } from '@automattic/shopping-cart';
+import { HorizontalRule, Button } from '@wordpress/components';
+import styled from '@emotion/styled';
+
+/**
+ * Internal dependencies
+ */
+import { LineItem } from 'calypso/my-sites/checkout/composite-checkout/components/wp-order-review-line-items';
+import { DomainSuggestions } from 'calypso/../packages/data-stores/src';
+import { MarketplaceThemeType } from 'calypso/my-sites/plugins/marketplace';
+
+interface PropsForMarketplaceShoppingCart {
+	onAddDomainToCart: () => void;
+	selectedDomain: DomainSuggestions.DomainSuggestion;
+}
+
+const MarketplaceTitle = styled.h1`
+	font-size: 1.5em;
+`;
+
+const ShoppingCartTotal = styled.div< { theme?: MarketplaceThemeType } >`
+	display: flex;
+	justify-content: space-between;
+	width: 100%;
+	margin-bottom: 20px;
+	div {
+		font-size: 1.15rem;
+		font-weight: ${ ( { theme } ) => theme.weights.bold };
+	}
+`;
+
+const FullWidthButton = styled( Button )`
+	justify-content: center;
+	width: 100%;
+`;
+
+export default function MarketplaceShoppingCart(
+	props: PropsForMarketplaceShoppingCart
+): JSX.Element {
+	const { onAddDomainToCart, selectedDomain } = props;
+	const { responseCart, isLoading } = useShoppingCart();
+	const { products, sub_total_display } = responseCart;
+
+	return isLoading ? (
+		<></>
+	) : (
+		<>
+			<MarketplaceTitle>Your cart</MarketplaceTitle>
```

Comment:
Note missing `__()` here and in few other lines in this file.

Reply:
Thanks this should be fixed now 👍